docs/source/design/load.rst

```diff
@@ -27,56 +27,51 @@ Each builder class implements methods for the following behaviors:
 
 #. (optional) **Transform the given records list into whatever new record
    format the entity builder needs**.
-#. (soon to be optional) **Compose a unique keystring from the entity's primary
-   constituent elements**. This is used internally for caching identifiers
-   assigned by the target service (received from the target service as part of
-   the response for sending an entity payload) so that they can be repeatably
-   assigned to the same entities again. (Target services often don't do this
-   themselves.)
-#. **Build a message payload that will be sent to the server to populate a new
-   (or updated) entity of its type**. The unique keystrings generated in the
-   previous steps are passed to this method as well so that they can be
-   included as part of the payload if desired.
+#. **Compose the entity's uniquely identifying primary elements**. This is used
+   for querying identifiers assigned by the target service so that they can be
+   repeatably assigned to the same entities again. (Target services often don't
+   do this equivalent entity replacement themselves.)
```

Comment:
As far as I understand, the sequential operation of `get_key_components` and `query_target_ids` would also create new target service IDs when entities don't exist. The statement here sounds a bit like this operation is only for replicating what's in the target service.

Reply:
The target service itself creates new IDs. Querying just determines if one already exists.